api/approx_count_distinct.md

```diff
@@ -0,0 +1,55 @@
+---
+api_name: approx_count_distinct()
+excerpt: Aggregate data into a hyperloglog for approximate counting without having to specify the number of buckets.
+license: community
+toolkit: true
+topic: hyperfunctions
+tags: [hyperfunctions, approximate count distinct, distinct count, hyperloglog]
+api_category: hyperfunction
+toolkit-experimental: false
```

Comment:
```suggestion
api_experimental: true
```
Should this be true? I just checked and it looks to me like it's under `toolkit_experimental` schema?